frontend/test/metabase-smoketest/admin_setup.cy.spec.js

```diff
@@ -0,0 +1,643 @@
+        .type("14"); // *** This pulls up title with the ID you've typed in
+      cy.findByText("Add filter").click();
+      cy.findByText("Visualize").click();
+
+      cy.contains("Awesome Concrete Shoes");
+      cy.contains("Mediocre Wooden Bench").should("not.exist");
+    });
+
+    it("should hide a table as admin", () => {
+      cy.visit("/admin/datamodel/database/1/");
+
+      // Hide table
+
+      cy.findByText("Reviews")
+        .find(".Icon-eye_crossed_out")
+        .click({ force: true });
+
+      // Check table hidden on home page
+
+      cy.visit("/");
+
+      cy.contains(", " + USERS.admin.first_name);
+      cy.contains("A look at your People table");
+      cy.contains("A look at your Reviews table").should("not.exist");
+
+      // Check table hidden while browsing data
+
+      cy.visit("/browse/1");
+
+      cy.contains("Learn about our data");
+      cy.contains("Test Table");
+      cy.contains("Reviews").should("not.exist");
+
+      // Check table hidden in notebook editor
+
+      cy.findByText("Test Table").click();
+      cy.get(".Icon-notebook").click();
+
+      cy.findByText("Join data").click();
+
+      cy.contains("Test Table");
+      cy.contains("Reviews").should("not.exist");
+
+      cy.pause();
+    });
+
+    it("should see changes to visibility, formatting, and foreign key mapping as user", () => {
+      signOut();
+      signInAsNormalUser();
+      cy.visit("/");
+
+      // Check table names and visibility
+
+      cy.contains("A look at your People table");
+      cy.contains("A look at your Test Table table");
+      cy.contains("Reviews").should("not.exist");
+
+      // Check question names and descriptions
+
+      cy.findByText("Browse all items").click();
+
+      cy.contains("Orders, Count");
+      cy.contains("Orders, Count, Grouped by Created At (year)").should(
+        "not.exist",
+      );
+
+      cy.findByText("Test Question").click();
+      cy.get(".Icon-pencil").click();
+      cy.findByText("Edit this question").click();
+
+      cy.contains("Edit question");
+      cy.contains("Testing question description");
+
+      cy.findByText("Cancel").click();
+
+      // Check column names and visiblity
+
+      cy.findByText("Browse Data").click();
+      cy.findByText("Sample Dataset").click();
+      cy.findByText("Test Table").click();
+
+      cy.contains("Visualization");
+      cy.contains("Discount").should("not.exist");
+      cy.contains("Sale");
+      cy.contains("Created At").should("not.exist");
+
+      // Check column formatting
+
+      cy.contains("USD");
+
+      // Check column foreign key mapping
+
+      cy.contains("Awesome Concrete Shoes");
+      cy.contains("Mediocre Wooden Bench");
+      cy.get(".Table-ID")
+        .eq("1")
+        .contains("14")
+        .should("not.exist");
+    });
+  });
+
+  describe("permission changes reflected", () => {
+    beforeEach(signInAsNormalUser);
+
+    it("should check current permissions as use", () => {});
```

Comment:
What's this intended to do?

Reply:
line 555 should say "as user". Does that clarify?

Reply:
Oh sorry, I meant that this just look like an empty test so wasn't sure if it should be here or not.

Reply:
Ah, yes. I meant to put this here as well as a couple other blank tests just to outline what still needs to be done. My next push will not have contain any blank tests.